playground/frontend/lib/components/link_button/link_button.dart

```diff
@@ -16,47 +16,48 @@
  * limitations under the License.
  */
 
-import 'package:easy_localization/easy_localization.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_svg/svg.dart';
-import 'package:playground/src/assets/assets.gen.dart';
 import 'package:url_launcher/url_launcher.dart';
 
 class LinkButton extends StatelessWidget {
   final String iconPath;
   final String text;
   final String url;
+  final Color? color;
+  final bool showText;
 
   const LinkButton({
     required this.iconPath,
     required this.text,
     required this.url,
+    this.showText = true,
+    this.color,
   });
 
-  factory LinkButton.colab(String url) {
-    return LinkButton(
-      iconPath: Assets.colab,
-      text: 'intents.playground.openGoogleColab'.tr(),
-      url: url,
-    );
-  }
-
-  factory LinkButton.github(String url) {
-    return LinkButton(
-      iconPath: Assets.github,
-      text: 'intents.playground.viewOnGithub'.tr(),
-      url: url,
-    );
-  }
-
   @override
   Widget build(BuildContext context) {
-    return TextButton.icon(
-      icon: SvgPicture.asset(iconPath),
-      onPressed: () {
-        launchUrl(Uri.parse(url));
-      },
-      label: Text(text),
+    final icon = SvgPicture.asset(
+      iconPath,
+      color: color,
     );
+    void onTap() => launchUrl(Uri.parse(url));
+    
+    if (showText) {
+      return TextButton.icon(
+        icon: icon,
+        onPressed: onTap,
+        label: Text(text),
+      );
+    } else {
```

Comment:
No `else` after `return`.

Reply:
I think that the importance of these branches is equal and I would like to emphasize this with the syntax